create-resources-at-scale/create-resources-at-scale.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Passed validation in Azure Cloud Shell Bash environment on 4/25/2022
+
+# Variable block
+# These variables have placeholder values that are replaced with values from the csv input file.
+Identifier="0"

```

Comment:
This is your CSV line number, correct? 

When creating a resource group, you'll need

```
let "randomIdentifier=$RANDOM*$RANDOM"
location="eastus"
resourceGroup="msdocs-script-rg-$randomIdentifier"
az group create --name $resourceGroup --location $location
```